### DISCUSSION TITLE
Archives, Storage, etc.

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1928] [Support] [answered]
Just wondering what the plans are for long-term archival and storage of Pubs. CLOCKSS, LOCKSS, etc. What are PubPub's plans?

**Comment by gabestein** [comment, marked as answer]
Long story short: we've talked with both Portico and CLOCKSS, and if you have a relationship with them, we can support them (although CLOCKSS will need some startup funding to build a scraper for us). But both require that publishers become members, and we both don't look like publishers to them, and don't want to assume that relationship for folks who are publishing.

We're still looking for solutions for long-term archival for all released Pubs, but we're committed to figuring it out. We like what DOAJ is doing, and think something like that could be an option. We've had a few contacts with IA. And we're planning to make it easy to push an entire community to the Underlay (which is nearing a first release), which will make it easier to retrieve structured versions for self-archiving/storage.

We also, of course, welcome ideas and connections in this area.

**Comment by writingpoli** [comment]
This question actually stems from a conversation I had with someone at CLOCKSS months and months ago (maybe over a year ago now). Their membership fee was too high for us right now. They suggested checking in on your plans -- so this is me finally getting around to it.

I've done a little bit of uploading our stuff to IA, but it's a big job for one person. I just don't want our contributors' work to be lost if I'm hit by a truck. :)

------- Original Message -------

**Reply by gabestein** [reply]
Gotcha. Probably worth us having another conversation with them, then, to see if we can do something along the lines of what DOAJ is doing to help bring the cost down for smaller folks.

**Comment by tosteiner** [comment]
Just thinking out loud here, I wonder if [Zenodo](https://zenodo.org) might be an interesting venue to consider?

Github, for example, has an automated way to archive DOI-citable repository snapshots on Zenodo (we've made [use of that option for the Open Science MOOC a couple of years ago](https://genr.eu/wp/cite/)), so might maybe also be possible for PubPub to implement such a route?

**Reply by gabestein** [reply]
Oh, that's very cool, thanks for sending it! We'll reach out to them and see about what an integration might look like. Might even be possible through their API directly, browsing docs.
